src/redux/actions/genes/index.js

```diff
@@ -2,10 +2,12 @@ import loadGeneExpression from './loadGeneExpression';
 import changeGeneSelection from './changeGeneSelection';
 import loadPaginatedGeneProperties from './loadPaginatedGeneProperties';
 import loadMarkerGenes from './loadMarkerGenes';
+import setGeneOrder from './setGeneOrder';
 
 export {
   loadGeneExpression,
   changeGeneSelection,
   loadPaginatedGeneProperties,
   loadMarkerGenes,
+  setGeneOrder,
```

Comment:
as I said elsewhere, genes store doesn't sound like the correct place to store the order of the genes in the Plots and Tables marker heatmap, consider moving it out.